### PR TITLE
Add link to video by TJ DeVries

### PR DESCRIPTION
**Pull request by rodrigc**
@tjdevries made a nice video about https://github.com/numToStr/Comment.nvim

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -428,6 +428,10 @@ require('Comment.utils').cmotion.{line,char,v,V}
 
 There are multiple ways to contribute reporting/fixing bugs, feature requests. You can also submit commentstring to this plugin by updating [ft.lua](./lua/Comment/ft.lua) and sending PR.
 
+### 📺Videos
```
Comment (numToStr) [inline README.md:431]:
```suggestion
### 📺 Videos
```

**Review by numToStr — CHANGES_REQUESTED** [review]

**Review by numToStr — APPROVED** [review]
Thanks 👍

**Comment by rodrigc** [comment]
Thanks, great plugin!  I wouldn't have known about it without the video from @tjdevries.

**Comment by tjdevries** [comment]
<3

**Pull request merged**